sdk/core/src/CryptoLib/lib/browser.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  ENCRYPT_OP,
+  DECRYPT_OP,
+  SIGN_OP,
+  VERIFY_OP,
+  AES_BROWSER_ALGO,
+  AES_LENGTH,
+  HMAC_BROWSER_ALGO,
+  HMAC_BROWSER,
+  HMAC_LENGTH,
+  SHA256_BROWSER_ALGO,
+  SHA512_BROWSER_ALGO,
+} from '../constants'
+import { arrayToBuffer } from '../helpers'
+import { fallbackRandomBytes } from './fallback'
+
+export function getBrowerCrypto(): Crypto {
```

Comment:
Is this a typo? Should it be getBrowserCrypto?

Reply:
Function removed entirely. Changes made to react-native sdk only